test/hmc_core.jl/gdemo_nuts.jl

```diff
@@ -44,7 +44,7 @@ for test_id = 1:2
 
     totla_num = 10000
     for iter = 1:totla_num
-      
+      global logϵ, lj_func, grad_func, M_adapt, δ, μ
```

Comment:
?

Reply:
This was not working when I copied and pasted the code into REPL. The `global` here is needed in Julia 1.0 because the global scoping rules changed. This is a probably skipped file in the tests iiuc, but I am not sure.